partner-chains-cli/src/deregister/mod.rs

```diff
@@ -0,0 +1,82 @@
+#[cfg(test)]
+mod tests;
+
+use crate::config::config_fields::{
+	CARDANO_COLD_VERIFICATION_KEY_FILE, CARDANO_PAYMENT_SIGNING_KEY_FILE,
+};
+use crate::config::{get_cardano_network_from_file, CHAIN_CONFIG_FILE_PATH, PC_CONTRACTS_CLI_PATH};
+use crate::io::IOContext;
+use crate::pc_contracts_cli_resources::establish_pc_contracts_cli_configuration;
+use crate::{smart_contracts, CmdRun};
+use anyhow::anyhow;
+
+#[derive(Debug, clap::Parser)]
+pub struct DeregisterCmd;
+
+impl CmdRun for DeregisterCmd {
+	fn run<C: IOContext>(&self, context: &C) -> anyhow::Result<()> {
+		let (chain_config, cardano_network) = read_chain_config_file(context)?;
+		context.print(
+		  &format!("This wizard will remove the candidate identified by given from the committee candidates of a chain identified by parameters:\n{}.\nCommittee Candidate Validator Address is {}\n",
```

Comment:
```suggestion
		  &format!(This wizard will remove the specified candidate from the committee candidates based on the following parameters:\n{}.\nValidator Address of the candidate: {}\n",
```

Reply:
I used almost this text - not committing suggestion, because it won't compile.